src/NzbDrone.Console/NzbDrone.Console.csproj

```diff
@@ -54,7 +54,7 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <PropertyGroup>
-    <ApplicationIcon>Radarr.ico</ApplicationIcon>

```

Comment:
Why was this renamed?

Reply:
just to this in line with the other forks